### ISSUE TITLE
PVC stuck in pending w/o any error

### ISSUE THREAD

**Issue by cvieri** [kubernetes/kubernetes#54231]
BUG REPORT
/sig storage 

PVC is stuck in pending state after configuring pvc+rbd

Expected to have any meaningful message

```
> apiVersion: storage.k8s.io/v1
> kind: StorageClass
> metadata:
>    name: slow
> provisioner: kubernetes.io/rbd
> parameters:
>     monitors: 127.0.0.1:6789
>     adminId: admin
>     adminSecretName: ceph-secret-admin
>     adminSecretNamespace: "kube-system"
>     pool: kube
>     userId: kube
>     userSecretName: ceph-secret-user
```

```
> {
>   "kind": "PersistentVolumeClaim",
>   "apiVersion": "v1",
>   "metadata": {
>     "name": "claim1"
>   },
>   "spec": {
>     "accessModes": [
>       "ReadWriteOnce"
>     ],
>     "resources": {
>       "requests": {
>         "storage": "3Gi"
>       }
>     },
>     "storageClassName": "slow"
>   }
> }
```

```
> root@ubuntu:~# kubectl describe pvc
> Name:          claim1
> Namespace:     default
> StorageClass:  fast
> Status:        Pending
> Volume:        
> Labels:        <none>
> Annotations:   volume.beta.kubernetes.io/storage-provisioner=kubernetes.io/rbd
> Capacity:      
> Access Modes:  
> Events:        <none>
> 
```

I am using kubeadm and

```
Client Version: version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.1", GitCommit:"f38e43b221d08850172a9a4ea785a86a3ffa3b3a", GitTreeState:"clean", BuildDate:"2017-10-11T23:27:35Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.6", GitCommit:"4bc5e7f9a6c25dc4c03d4d656f2cefd21540e28c", GitTreeState:"clean", BuildDate:"2017-09-14T06:36:08Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by dixudx** [comment]
> root@ubuntu:~# kubectl describe pvc
> Name:          claim1
> Namespace:     default
> StorageClass:  fast
> Status:        Pending

@cvieri Describing PVC, the storageclass is `fast` instead of `slow`. Please check your settings.

Also this event issue has been fixed by @jsafrane in PR #53322.

**Comment by cvieri** [comment]
@dixudx the problem was related to ceph cluster being in error state. Can it be somehow signalized in kubectl describe?
